packer/builder/azure/driver_powershell/builder.go

```diff
@@ -80,6 +83,8 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 		"location":                  &b.config.Location,
 		"instance_size":             &b.config.InstanceSize,
 		"user_image_label":          &b.config.UserImageLabel,
+		"vnet":                      &b.config.VNet,
+		"subnet":                    &b.config.Subnet,
```

Comment:
The implementation for the powershell builder seems partial. Either provide a full implementation or a REST-api only version... :-)